src/components/Footer/Footer.jsx

```diff
@@ -165,21 +165,21 @@ const Footer = ({ strings }) => (
               href="https://play.google.com/store/apps/details?id=com.opendota.mobile&hl=en"
               style={{ position: 'relative', left: '13px', top: '12px' }}
             >
-              <img src="/assets/images/google_play_store.png" alt="" height="46px" />
+              <img src="/assets/images/google_play_store.png" alt="download the android app on google play store" height="46px" />
             </a>
             <a
               href="https://itunes.apple.com/us/app/opendota/id1354762555?ls=1&mt=8"
               style={{ position: 'relative', left: '20px', top: '5px' }}
             >
-              <img src="/assets/images/apple_app_store.png" alt="" height="31px" />
+              <img src="/assets/images/apple_app_store.png" alt="download the iOS app on the app store" height="31px" />
```

Comment:
Functional links should meet [W3 accessibility standards](https://www.w3.org/WAI/tutorials/images/functional/) by providing alt text that accurately describes what the link does. 

<details><summary>Current behavior</summary>
<img width="949" alt="Screen Shot 2021-10-02 at 11 15 47 AM" src="https://user-images.githubusercontent.com/15097156/135724645-45ab0b7c-3e9f-4087-a184-dcc7e66e3d89.png">
</details>

<details><summary>With updated alt text</summary>
<img width="825" alt="Screen Shot 2021-10-02 at 11 21 07 AM" src="https://user-images.githubusercontent.com/15097156/135724651-4489f57d-fe63-4154-a57d-fcbe8f7d5411.png">

</details>